### ISSUE TITLE
accessibility: mui react table missing semantic

### ISSUE THREAD

**Issue by fcnjd** [marmelab/react-admin#11108]
**What you were expecting:**
As a blind user relying on screenreader and Braille display, I expect clickable elements to be shown as either links or buttons.

**What happened instead:**
Clickable table rows in MUI react tables are shown as normal text, although they do execute an action when enter is pressed on them.

**Steps to reproduce:**
1. Go to the helpdesk demo on (https://marmelab.com/react-admin-helpdesk/#/tickets)
2. Turn on a screenreader, tested with NVDA on Windows
3. Navigate the tickets table
4. Find that entries are neither links nor buttons, but execute when pressed

**Other information:**
See this related parent issue on Synapse Admin:
https://github.com/etkecc/synapse-admin/issues/724

**Comment by slax57** [comment]
confirmed, thanks!

**Comment by fzaninotto** [comment]
Thanks for the heads up. Would you mind sharing your preferred solution? It looks that Synapse Admin's solution (adding an `aria-label` and a `tabindex="0"` isn't really enough, is it.

**Comment by fcnjd** [comment]
@fzaninotto Difficult to say. With the current latest docker image of Ketesa, speech somehow announces link start and end, but I don't really get it shown on my braille display, and it's not in the NVDA element list where it is supposed to be, so yes, it doesn't look like enough.
